sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_usage_v1/schema.yaml

```
@@ -20,6 +20,16 @@ fields:
   type: STRING
   description: The type of the job
 
+- mode: NULLABLE
+  name: reservation_id
+  type: STRING
+  description: Name of the primary reservation (slot allocation) assigned to this job
```

Would this description provide a quick understanding for those who are not familiar with the concept of reservations? Feel free to adjust it,
```suggestion
  description: Name of the primary reservation assigned to this job. Reservations are BigQuery slots allocated to projects in order to distribute the resources and processing capacity.
```

---

Thanks for the suggestion.  This makes it more clear.